tools/querytee/proxy_endpoint.go

```diff
@@ -182,6 +209,22 @@ func (p *ProxyEndpoint) executeBackendRequests(req *http.Request, resCh chan *ba
 				"query", query,
 				"user", req.Header.Get("X-Scope-OrgID"),
 				"err", err,
+				"expected-response-duration", expectedResponse.elapsedTime,
+				"actual-response-duration", actualResponse.elapsedTime,
+			)
+		}
+
+		// Log queries that are slower in some backends than others
+		if p.slowResponseThreshold > 0 && slowestDuration-fastestDuration >= p.slowResponseThreshold {
+			level.Warn(p.logger).Log(
+				"msg", "response time between backends exceeded threshold",
+				"route-name", p.routeName,
```

Comment:
Isn't our convention to snake case log message fields?
```suggestion
				"route_name", p.routeName,
```

Reply:
route-name is consistent with the pre-existing log (currently line 208). I see we have a mix within the code, even some camelCase (particularly where it is expressing a variable). I'll change them all to snake_case, but it may break people who are querying/filtering off these logfmt keys. (That said, we make no guarantees on log formats)